unicorn/CMakeLists.txt

```diff
@@ -1,15 +1,37 @@
 INCLUDE_DIRECTORIES(
     ${PROJECT_SOURCE_DIR}/unicorn/include)
 
+# Headers are not required but can be usefull in some cases,
+# for example in some IDEs it's required to add files to project.
+SET(HEADERS
```

Comment:
An IDEs can be configured other way. QtCreator, for example, allows to create generic project, where you can list all files it have to include in the project. If CLion doesn't allow this, open an issue [here](https://youtrack.jetbrains.com/issues/CPP).